src/providers/auth.ts

```diff
@@ -52,21 +54,9 @@ export function _userWithOptions(options: DeploymentOptions) {
   }, options);
 }
 
-export class UserRecordMetadata implements firebase.auth.UserMetadata {
```

Comment:
I think it's possible that we break some user's code if we remove exported class like this one (similarly for any other exported objects)

Let's create an alias to prevent breaking change (and maybe add a note in go/cf3-v4-sdk to clean it up in the next major version.

Reply:
Oh yeah that's a great point. I re-exported them in the auth code path, added a todo comment about yanking them, and updated that doc.